code/modules/cargo/order.dm

```diff
@@ -21,10 +21,13 @@
 
 	if(prob(MANIFEST_ERROR_CHANCE))
 		errors |= MANIFEST_ERROR_NAME
+		investigate_log("Supply order #[order_id] generated a manifest with the incorrect station name.", INVESTIGATE_CARGO)
 	if(prob(MANIFEST_ERROR_CHANCE))
 		errors |= MANIFEST_ERROR_CONTENTS
+		investigate_log("Supply order #[order_id] generated a manifest missing listed contents.", INVESTIGATE_CARGO)
 	if(prob(MANIFEST_ERROR_CHANCE))
 		errors |= MANIFEST_ERROR_ITEM
+		investigate_log("Supply order #[order_id] generated with missing contents", INVESTIGATE_CARGO)
```

Comment:
```suggestion
		investigate_log("Supply order #[order_id] generated with incorrect contents", INVESTIGATE_CARGO)
```

Both so it doesn’t look like some bug error and because there might have contents in it that are just wrong.